applications/serial_lte_modem/Kconfig

```
@@ -81,8 +81,16 @@ config SLM_START_SLEEP
 	help
 	  Put nRF9160 into deep sleep mode after startup.
 
-config SLM_INTERFACE_PIN
-	int "Interface GPIO to wake up or exit idle mode"
+config SLM_WAKEUP_PIN
+	int "Interface GPIO to exit from sleep or idle mode"
```

Why is there no default here ?
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#redundant-defaults
> Defaults should always be given for int and hex symbols, however, as they implicitly default to the empty string. 

Also, the prompt text looks more like help text than a title.
Proposal:
```suggestion
	int "SLM wakeup pin"
	help
	  Interface GPIO pin used by Serial LTE Modem to exit from sleep or idle mode
```

---

you missed this part of my comment:
> Why is there no default here ?
> https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#redundant-defaults
>
> > Defaults should always be given for int and hex symbols, however, as they implicitly default to the empty string.

the `int` setting doesn't have a default, which means it becomes an empty string which has the potential to cause spurious errors in code:
https://github.com/nrfconnect/sdk-nrf/blob/a63ce76f1205db154ca54eb703fc2e690ca2dde6/applications/serial_lte_modem/Kconfig#L84-L87

Please give an explanation and comment in code if there is a special reason this should not have a default.

---

Sorry for overlooking. We have configured the value in Zephyr project file.
But now I added -1 as default and also Build check that this value must be equal or greater than 0.
Thanks for pointing it out